### PR TITLE
Collect context and pass it to the core

### PR DESCRIPTION
**Pull request by bugraderre**
This commit implements the following features:
- the Flask contribution collects the application service's context from
the Flask configuration and passes it to the core context

What this commit not does:
- it is not using or processing the collected context

Resolves: #12 

Signed-off-by: Bugra Derre <bugra.derre@proofdock.io>

### CODE REVIEW

File: src/contrib-flask/README.md [hunk 1]
```diff
@@ -25,7 +25,13 @@ To be defined ...
 
 ## Configuration
 
-To be defined ...
+Configure your Flask configuration with the following properties:
+
+| Property | Flask configuration | Description |
+| ---      | ---                 | ---         |
+| `PROOFDOCK_SERVICE_NAME` | `app.config.setdefault('PROOFDOCK_SERVICE_NAME', '')` | Set up a service application name for your Flask app |
+| `PROOFDOCK_SERVICE_ENVIRONMENT` | `app.config.setdefault('PROOFDOCK_SERVICE_ENVIRONMENT', '')` | The environment in which your service application is running |
```
Comment (mkaszub) [inline src/contrib-flask/README.md:33]: What is service environment for? There is already in flask environment variable called: FLASK_ENV

Comment (bugraderre) [reply src/contrib-flask/README.md:33]: service environment := the stage or environment where the Flask application is running.

> Good hint. Let's use the FLASK_ENV then instead creating a redundant one and using our own

Comment (mkaszub) [reply src/contrib-flask/README.md]: Probabu it is a mistake. Flask set boths as i remember correctly...

Please check this out:
https://flask.palletsprojects.com/en/1.1.x/config/#environment-and-debug-features

Comment (mkaszub) [reply src/contrib-flask/README.md]: Ok lets leave it as is for now... it may not fit the best our use case...

**Pull request merged**